xain_fl/grpc/test_coordinator.py

```diff
@@ -196,3 +234,42 @@ def test_remove_participant():
     coordinator.on_message(coordinator_pb2.RendezvousRequest(), "participant1")
 
     assert coordinator.state == coordinator_pb2.State.ROUND
+
+
+def test_number_of_selected_participants():
+    # test that the coordinator needs minimum 3 participants and selects 2 of them
+    coordinator = Coordinator(
+        minimum_participants_in_round=2, fraction_of_participants=0.6
+    )
+    coordinator.on_message(coordinator_pb2.RendezvousRequest(), "participant1")
+    coordinator.on_message(coordinator_pb2.RendezvousRequest(), "participant2")
+
+    # the coordinator should not start a round but wait for a third
+    assert coordinator.minimum_connected_participants == 3
+    assert coordinator.state == coordinator_pb2.State.STANDBY
```

Comment:
can also check this after the first connects

Reply:
thanks, change